src/pages/Account/AccountPage.module.scss

```diff
@@ -164,6 +164,20 @@
   }
 }
 
+.prefix {
+  position: absolute;
+  color: var(--jungle-green);
+  font-size: 14px;
+  line-height: 20px;
+  top: 50%;
+  transform: translateY(-50%);
+  left: 25px;
+
+  @include responsive('phone', 'phone-xs') {
```

Comment:
Better to use the new `.phone, .phone-xs` classes

Reply:
Can you please describe more? Hoe can we use them?

Reply:
@peratik 
```scss
:global(.phone) &, :global(.phone-xs) & {
    left: 10px;
}
```